apps/web/app/[locale]/auth/team/component.tsx

```
@@ -131,6 +133,61 @@
+						onError={() => handleCaptchaError()}
+					/>
+				);
+			case 'cloudflare':
+				return (
+					<Turnstile
+						sitekey={process.env.NEXT_PUBLIC_TURNSTILE_SITE_KEY ?? ''}
+						onSuccess={(token) => handleCaptchaVerify(token)}
+						onError={() => handleCaptchaError()}
+					/>
+				);
+			case 'google':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)